src/Player.hpp

```diff
@@ -0,0 +1,34 @@
+#include <string>
+#include <vector>
+#include <memory>
+#include "Property.hpp"
+class Player{
+
+    std::string color_;
+    int money_;
+    int location_;
+    std::vector<Property> properties_;
```

Comment:
It's good to define a collection of Properties in Property.hpp like `using Properties = std::vector<Property>`.
The same if you'll need let's say shared_ptr<Property> in a future, you could add `using PropertyPtr = std::shared_ptr<Property>;`